components/Avatar/avatar.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar</title>
+    <link rel="stylesheet" href="./avatar.css" />
+  </head>
+  <body>
+    <main class="container">
+      <div class="leftNav">
+        <div class="content-header">
+          <h2 class="heading-main">Avatar</h2>
+        </div>
+        <div class="content-description">
+          <p>
+            Avatars are images that users can set as their profile picture.These
+            can be custom photos uploaded by users, or generated as Identicons
+            as a placeholder.
+          </p>
+        </div>
+      </div>
+
+      <div class="content">
+        <h2 class="heading">Basic Example</h2>
+
+        <p>Add .avatar to any <img /> element to make it an avatar.</p>
+
+        <img class="avatar" src="../../assets/my-img.jpg" alt="my-profile" />
+
+        <xmp>
+            <img class="avatar" src="../../assets/my-img.jpg" alt="my-profile" />
+        </xmp>
+
+        <hr />
+
+        <h2 class="heading">Small Avatar Example</h2>
+
+        <p>
+          Anything less than equal 30px wide should include the .avatar-small
+          modifier class to reset the border-radius to a more appropriate level.
+        </p>
+
+        <img
+          class="avatar-small"
+          src="../../assets/my-img.jpg"
+          alt="my-profile"
+        />
+
+        <xmp>
```

Comment:
Looks like xmp is deprecated.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/xmp

Reply:
removed xmp and used pre tag thanks for the link.